plantnet 404 species not found

check following 500 case while querying plantnet :

```
2020-05-03T18:48:06.085402+00:00 app[web.1]: 2020-05-02 10:52:21 --- @CometMarie: Quelqu'un sait identifier cette plante? 🤔 https://t.co/2FvbkuSc4y
2020-05-03T18:48:06.085402+00:00 app[web.1]: first media url : https://pbs.twimg.com/media/EXAdvCAWsAAA9D2.jpg
2020-05-03T18:48:06.085403+00:00 app[web.1]:
2020-05-03T18:48:06.085511+00:00 app[web.1]: [2020-05-03 18:48:06.085] [DEBUG] PlantnetBTP - candidateImage : https://pbs.twimg.com/media/EXAdvCAWsAAA9D2.jpg
2020-05-03T18:48:06.085592+00:00 app[web.1]: [2020-05-03 18:48:06.085] [INFO] MyPlantnetClient - identify following image : https://pbs.twimg.com/media/EXAdvCAWsAAA9D2.jpg
2020-05-03T18:48:06.800315+00:00 heroku[router]: at=info method=GET path="/hook" host=botentrain.herokuapp.com request_id=1d8332fb-90dd-48e2-9942-eb7b083be266 fwd="116.203.134.67" dyno=web.1 connect=1ms service=1218ms status=500 bytes=312 protocol=https
2020-05-03T18:48:06.795845+00:00 app[web.1]: [2020-05-03 18:48:06.795] [ERROR] MyPlantnetClient - Pla@ntnet identify error (404) Not Found - details:{"statusCode":404,"error":"Not Found","message":"Species not found"}
2020-05-03T18:48:06.795946+00:00 app[web.1]: [2020-05-03 18:48:06.795] [ERROR] PlantnetBTP - Pla@ntnet identify error (404) Not Found

```

--- a/core/BotEngine.js
+++ b/core/BotEngine.js
@@ -57,7 +57,7 @@ class BotEngine {
     plugin.process({ "doSimulate" : doSimulate }, (err, result) => {
         if(err) {
           engine.logger.warn("plugin response status:" + err.status + " msg:" + err.message);
-          engine.newsBot.add(err.message);
+          engine.newsBot.add(err.html ? err.html : err.message);
           cb(err);
           return;
         }

--- a/core/MyPlantnetClient.js
+++ b/core/MyPlantnetClient.js
@@ -53,7 +53,7 @@ class MyPlantnetClient {
           let errDetails = err.response.text;
           let errResult = "Pla@ntnet identify error (" + errStatus + ") " + errError;
           this.logger.error(errResult + " - details:" + errDetails);
-          cb(errResult);
+          cb({ message: errResult, status: errStatus });
           return;
         }
         cb(false, res.body);

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "botEnTrain",
-  "version": "1.1.8",
+  "version": "1.1.9",
   "description": "Twitter botEnTrain",
   "main": "bet.js",
   "scripts": {

--- a/plugins/PlantnetBTP.js
+++ b/plugins/PlantnetBTP.js
@@ -90,8 +90,16 @@ class PlantnetBTP {
 
         this.plantnetClient.identify(candidateImage, PLANTNET_SIMULATE, (err, plantResult) => {
             if (err) {
-                this.logger.error(err);
-                cb({"message": "impossible d'identifier l'image", "status": 500});
+                this.logger.error(err.message);
+                if (err.status && err.status == 404) {
+                    this.replyNotFoundResult(doSimulate, pluginTags, tweetCandidate, cb);
+                    return;
+                }
+                cb({"message": "impossible d'identifier l'image",
+                    "html": "<b>Tweet</b>: <div class=\"bg-warning\">" +
+                    this.twitterClient.tweetHtmlOf(tweetCandidate) + "</div>" +
+                    " <b>Erreur</b>: impossible d'identifier l'image",
+                    "status": 500});
                 return;
             }
 
@@ -117,33 +125,27 @@ class PlantnetBTP {
       (illustrateImage ? "\n\n" + illustrateImage : "") + "\n\n" +
       pluginTags;
 
-      this.replyTweet(doSimulate, tweetCandidate, replyMessage, (err, replyTweet) => {
-          if (err) {
-              this.logger.error(err);
-              cb({"message": "impossible de répondre au tweet", "status": 500});
-              return;
-          }
-          cb(false, {
-              "html": "<b>Tweet</b>: <div class=\"bg-info\">" +
-                  this.twitterClient.tweetHtmlOf(tweetCandidate) + "</div>" +
-                  "<b>Réponse émise</b>: " +
-                  this.twitterClient.tweetHtmlOf(replyTweet),
-              "text": "\nTweet:\n\t" +
-                  this.twitterClient.tweetLinkOf(tweetCandidate) + "\n\t" +
-                  this.twitterClient.tweetInfoOf(tweetCandidate) + "\n" +
-                  "Reply sent:\n\t" +
-                  this.twitterClient.tweetLinkOf(replyTweet) + "\n\t" +
-                  this.twitterClient.tweetInfoOf(replyTweet) + "\n"
-          });
-      });
+      this.replyResult(doSimulate, tweetCandidate, replyMessage, cb);
     });
   }
+
+  replyNotFoundResult(doSimulate, pluginTags, tweetCandidate, cb) {
+      let replyMessage = "Bonjour, j'ai interrogé Pl@ntnet pour tenter d'identifier votre première image" +
+      " mais a priori il ne s'agit ni d'une plante ni d'une fleur 😏 ?\n" +
+      "Je me suis bien fait avoir 😊 !\n\n" +
+      pluginTags;
+      this.replyResult(doSimulate, tweetCandidate, replyMessage, cb);
+   }
 
   replyNoScoredResult(doSimulate, pluginTags, tweetCandidate, cb) {
       let replyMessage = "Bonjour, j'ai interrogé Pl@ntnet pour tenter d'identifier votre première image" +
       " mais cela n'a pas donné de résultat concluant 😩 (score>" + PLANTNET_MINIMAL_PERCENT + "%).\n" +
       "Astuce: bien cadrer la fleur ou feuille\n\n" +
       pluginTags;
+      this.replyResult(doSimulate, tweetCandidate, replyMessage, cb);
+  }
+
+  replyResult(doSimulate, tweetCandidate, replyMessage, cb) {
       this.replyTweet(doSimulate, tweetCandidate, replyMessage, (err, replyTweet) => {
           if (err) {
               this.logger.error(err);

